.github/workflows/sync-with-upstream.yaml

```diff
@@ -59,23 +59,23 @@ jobs:
             --exclude .git/ --exclude .github/workflows/.template_version template/ repo/
 
           # Check if the branch exists in the current repo.
-          cd repo/
-          if [ $(git rev-parse --verify origin/sync-$LATEST_VERSION 2>/dev/null) ]; then
+          cd repo/ || exit 1
+          if [ "$(git rev-parse --verify origin/sync-"""$LATEST_VERSION""" 2>/dev/null)" ]; then
             echo "Branch sync-$LATEST_VERSION already exists. Exiting..."
             exit 0
           fi
 
           # Create a branch, commit, and push the changeset.
-          git checkout -b sync-$LATEST_VERSION
-          echo $LATEST_VERSION > .github/workflows/.template_version
+          git checkout -b sync-"$LATEST_VERSION"
+          echo "$LATEST_VERSION" > .github/workflows/.template_version
           git add .
-          git config --global user.name $USER_NAME
-          git config --global user.email $USER_EMAIL
+          git config --global user.name "$USER_NAME"
+          git config --global user.email "$USER_EMAIL"
           git config --list --global # For debug purposes.
           git commit -m "chore: sync with template"
```

Comment:
While we’re making changes here, can we change `-m` to `--message` as well, please?

Reply:
See commit [3dd239a](https://github.com/jenstroeger/python-package-template/pull/505/commits/3dd239a6f2cec3327a94ff788e658632b6a1f5d6).